custom/lib/Authentication.php

```diff
@@ -2,99 +2,46 @@
      *
      * @param Passage $passage
      *
      * @throws \InvalidArgumentException
      */
-    public function __construct(Passage $passage)
-    {
-        $this->passage = $passage;
-
-        $appId = $this->passage->getAppId();
-        $jwtIssuer = 'https://auth.passage.id/v1/apps/' . $appId . '/.well-known/jwks.json';
-
-        $this->jwks = $this->fetchJWKS($jwtIssuer);
-    }
-
-    /**
-     * Returns the JWKS for the current app
-     *
-     * @param string $url
-     *
-     * @return CachedKeySet UserId of the Passage user
-     */
-    private function fetchJWKS(string $url): CachedKeySet
+    public function __construct(private Passage $passage)
     {
-        $httpClient = new Client();
-        $httpFactory = new HttpFactory();
-
-        $cacheItemPool = CacheManager::getInstance('files');
+        trigger_error('Method ' . __METHOD__ . ' is deprecated', E_USER_DEPRECATED);
 
-        $keySet = new CachedKeySet(
-            $url,
-            $httpClient,
-            $httpFactory,
-            $cacheItemPool,
-            null,
-            true
-        );
-
-        return $keySet;
+        $this->passage = $passage;
     }
 
     /**
-     * Determine if the provided token is valid when compared with its
-     * respective public key.
+     * @deprecated 1.0.0
+     * @see Passage->auth->validateJwt
      *
-     * @param string $jwtString Authentication token
+     * Validate that a JWT is valid and return the Passage user ID associated with the token.
      *
-     * @return string sub claim if the jwt can be verified, or Error
+     * @param string $jwtString The authentication token to be validated
+     * @return string User ID of the Passage user
      */
     public function validateJWT(string $jwtString): string | null
     {
-        try {
-            $decodedHeader = JWT::urlsafeB64Decode(explode('.', $jwtString)[0]);
-            $header = json_decode($decodedHeader);
-
-            $kid = $header->kid;
-
-            if (!$kid) {
-                throw new ApiException(
-                    'Could not verify token: Missing kid in token',
-                    401
-                );
-            }
+        trigger_error('Method ' . __METHOD__ . ' is deprecated', E_USER_DEPRECATED);
 
-            $decodedToken = JWT::decode($jwtString, $this->jwks);
-            $userID = $decodedToken->sub;
-
-            if ($userID) {
-                return strval($userID);
-            } else {
-                throw new ApiException(
-                    'Could not verify token: Could not retrieve user id',
-                    401
-                );
-            }
-        } catch (\Exception $e) {
+        try {
+            return $this->passage->auth->validateJwt($jwtString);
+        } catch (Exception $e) {
             throw new ApiException(
-                'Could not verify token: ' . $e->getMessage(),
+                "Could not verify token: Could not verify token: {$e->getMessage()}",
```

Comment:
Why is the prefix repeated in this change?

Reply:
This was the exact message that it previously emitted since it throws exceptions in the same `try` block which are all prefixed with "Could not verify token: "

Reply:
couldn't repro the issue, but just removed it anyways in dbd62485e40784cd69f9123ef66fddf4df4ea364